client/src/components/modals/nr-not-required.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <v-dialog v-model="showModal" max-width="40%">
+  <!-- NB: dummy modal attach point so unit tests can see it -->
+  <v-dialog v-model="showModal" max-width="40%" attach="">
```

Comment:
This seems to render the dialog in-line. Cool!

(We might not need to attach dialogs to their parent any more in our other UI projects.)

(Yes, I tested that this dialog still displays correctly in the running app.)

Reply:
Is there a default `attach` value within the vue component itself? Just curious how this would differ from not passing the property at all? Unless this is overriding a default..?

Reply:
The default is that it attaches to v-app.

With no attach prop, in the unit tests, you don't see the dialog contents (because it's rendered outside the component).

With my code change, it basically attaches to itself, so you can see the dialog's HTML in the unit test.

https://vuetifyjs.com/en/api/v-dialog/